amanuensis/resources/message/__init__.py

```diff
@@ -7,92 +7,133 @@
 from pcdcutils.environment import is_env_enabled
 
 from amanuensis.config import config
-from amanuensis.errors import NotFound, Unauthorized, UserError, InternalError, Forbidden
+from amanuensis.errors import (
+    NotFound,
+    Unauthorized,
+    UserError,
+    InternalError,
+    Forbidden,
+)
 from amanuensis.resources import userdatamodel as udm
+
 # from amanuensis.resources.userdatamodel import get_all_messages, get_messages_by_request, send_message
 from amanuensis.resources.fence import fence_get_users
 from amanuensis.auth.auth import current_user
+from flask import current_app
 from amanuensis.models import (
     ConsortiumDataContributor,
     Message,
-    Receiver
+    Receiver,
     # Request
 )
+from hubspot.crm.contacts import PublicObjectSearchRequest, ApiException
 
-# TODO initialize on app init instead of here
-from hubspotclient.client.hubspot.client import HubspotClient
 
 logger = get_logger(__name__)
 
 
+def get_contacts_by_committee(committee, desired_properties=["email"]):
```

Comment:
This didnt return an error but it did return [] using INRG. I looked through the set up in Hubspot and it doesn't seem like there is a disease group column